mkt/developers/forms.py

```diff
@@ -535,7 +540,7 @@ class AppFormBasic(AddonFormBase):
 
     class Meta:
         model = Webapp
-        fields = ('slug', 'manifest_url', 'description', 'tags')
+        fields = ('slug', 'manifest_url', 'hosted_url', 'description', 'tags')
```

Comment:
i think it's in order of the form fields, and the order of the form fields have meaning